apps/dashboard/src/components/template-store/workflow-card.tsx

```diff
@@ -19,19 +20,23 @@ export function WorkflowCard({
 }: WorkflowCardProps) {
   return (
     <Card
-      className="border-stroke-soft min-h-[138px] min-w-[250px] border shadow-none hover:cursor-pointer"
+      className="border-stroke-soft min-h-[120px] min-w-[250px] border shadow-none hover:cursor-pointer"
       onClick={onClick}
     >
       <CardContent className="p-3">
         <div className="overflow-hidden rounded-lg border border-neutral-100">
-          <div className="bg-bg-weak relative h-[114px] bg-[url(/images/dots.svg)] bg-cover">
+          <div className="bg-bg-weak relative h-[100px] bg-[url(/images/dots.svg)] bg-cover">
             <div className="flex h-full w-full items-center justify-center">
-              {steps.map((step, index) => (
-                <React.Fragment key={index}>
-                  <WorkflowStep step={step} />
-                  {index < steps.length - 1 && <div className="h-px w-6 bg-gray-200" />}
-                </React.Fragment>
-              ))}
+              {!steps?.length ? (
+                <RiAddFill className="text-[#D6D6D6]" />
+              ) : (
+                steps.map((step, index) => (
+                  <React.Fragment key={index}>
+                    <WorkflowStep step={step} />
+                    {index < steps.length - 1 && <div className="h-px w-6 bg-gray-200" />}
```

Comment:
What is this div for? Is it a separator?

Reply:
It's the connector line between the steps